gradle-sls-packaging/src/main/resources/go-java-version.properties

```diff
@@ -0,0 +1,3 @@
+# Version of https://github.com/palantir/go-java-launcher to use. Kept up-to-date by excavator.
+# If you move this file, please make sure to update the respective excavator.
+go-java-version=1.17.0
```

Comment:
should we just get people to declare the version of go-java-launcher in versions.props like we do for gofigure and sls-validation?

Reply:
Added in https://github.com/palantir/sls-packaging/pull/1294 for future reference

Reply:
FWIW you can set a default dependency to keep upgrades flowing and then roll out the versions.props change separately/async